fuzz/fuzz_targets/packet.rs

```diff
@@ -0,0 +1,12 @@
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+
+extern crate proto;
+use proto::fuzzing::{PacketParams, PartialDecode};
+
+fuzz_target!(|data: PacketParams| {
+    let len = data.buf.len();
+    if let Ok(decoded) = PartialDecode::new(data.buf, data.local_cid_len) {
+        assert_eq!(len, decoded.0.len());
```

Comment:
This only holds if `decoded.1` is `None`. If it's instead `Some(x)`, we should check against `decoded.0.len() + x.len()`.